pokete_classes/audio.py

```diff
@@ -1,20 +1,21 @@
 """This manages audio playback"""
 
 import multiprocessing
+from pathlib import Path
 try:
     from playsound import playsound
 except ModuleNotFoundError:
     from .dummy_playsound import playsound
 from .settings import settings
 
 
-MUSIC_PATH = __file__.replace("pokete_classes/audio.py", 'assets/music/')
+MUSIC_PATH = Path(__file__).parent.parent / 'assets' / 'music'
```

Comment:
just FYI, should be able to do `Path(__file__).parents[1]`

Reply:
Oh yes, that's a good tip.